docker-compose-lviv.yml

```diff
@@ -0,0 +1,56 @@
+version: "3"
+services:
+  db:
+    image: postgis/postgis:13-3.0
+    volumes:
+      - ./db:/var/lib/postgresql/data
+    expose:
+      - 5432
+    ports:
+      - "5433:5432"
+    environment:
+      - POSTGRES_DB=sip_db
+      - POSTGRES_USER=sip
+      - POSTGRES_PASSWORD=eiJ2koo6
+
+  webapplication:
+    build:
+      context: ./webapplication
+      dockerfile: Dockerfile
+    command: /bin/bash -c "exec invoke run-lviv"
+    environment:
+      - DJANGO_SETTINGS_MODULE=sip.settings-lviv
+      - POSTGRES_DB=sip_db
+      - POSTGRES_USER=sip
+      - POSTGRES_PASSWORD=eiJ2koo6
+    volumes:
+      - ./webapplication:/code
+      - /mnt/storage/geoap_data:/data
+      - /var/run/docker.sock:/var/run/docker.sock
+    depends_on:
+      - db
+
+  webviewer:
+    image: node:14.5.0
+    volumes:
+      - ./webviewer:/client
+    working_dir: /client
+    expose:
+      - 3000
+    ports:
+      - "3000:3000"
+    command: bash -c "npm install && npm start"
```

Comment:
as we made a custom docker-compose-lviv.yml lets get rid of node dev server, see how it is done in https://github.com/QuantuMobileSoftware/geoap/blob/master/docker-compose-prod.yml.